service.subtitles.napisy24pl/service.py

```diff
@@ -306,6 +89,7 @@ def get_params():
 
     return param
 
+
 params = get_params()
```

Comment:
Hm...
I don't see a reason for the source to **not** be encapsulated in either their own methods or the typical _python main function_ as shown above.

I just think it's a bad practice unless it is required.
But as I said, no need to change it ;)